### DISCUSSION TITLE
All information deleted

### DISCUSSION THREAD

**Discussion by 5tatic** [FlowiseAI/Flowise#1037] [Q&A] [answered]
I have a working chatfow, or i did....

Today I opened up the railway domain and there was just nothing there. 
No API keys I had linked 
It was just all gone. 

anyone have any idea how to recover deleted workflows. 
![image](https://github.com/FlowiseAI/Flowise/assets/130356809/be6dba87-cc79-482c-a1db-3d1e740894e2)

**Comment by yavisht** [comment, marked as answer]
@5tatic 

Railway will wipe data every time you redeploy unless you do the following:

### Step 1: Attach a volume and give it a path
<img width="631" alt="Screen Shot 2023-10-12 at 12 44 56 pm" src="https://github.com/FlowiseAI/Flowise/assets/6112201/bdf61055-5fd7-4923-9a71-6cc75f3247ed">

### Step 2: Now set the following environment variables
<img width="625" alt="Screen Shot 2023-10-12 at 12 42 42 pm" src="https://github.com/FlowiseAI/Flowise/assets/6112201/ea547c2e-d228-462d-976a-b98ffe8dc821">

**Comment by 5tatic** [comment]
Thank you I appreciate that.
________________________________
